Makefile

```diff
@@ -239,10 +239,10 @@ generate: ## Generate code
 
 .PHONY: generate-go
 generate-go: $(MOCKGEN)
-	go generate ./...
 	$(MAKE) generate-go-core
 	$(MAKE) generate-go-eks-bootstrap
 	$(MAKE) generate-go-eks-controlplane
+	go generate ./...
```

Comment:
Is there any reason to change it?

Reply:
this is to fix `make generate`.  Based on the [change](https://github.com/ykakarap/cluster-api-provider-aws/pull/1/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52) suggested by @randomvariable

Reply:
Yup.